docs/features/runs/2_Analyzing Runs/3_artifacts.md

```diff
@@ -94,3 +94,27 @@ grid artifacts runname-expN
 :::note
 If you have additional questions about Runs, visit the [FAQ](https://docs.grid.ai/features/runs/faq). The section is periodically updated this with common questions from the Grid community.
 :::
+
+
+### See where your artifacts are stored in the cloud (BYOC clusters only)
+
+If you launched a run on a [BYOC cluster](../../../platform/2_Custom%20Cloud%20Credentials/1_byoc.md)
+and would like to get the cloud storage location for the artifacts instead of downloading them,
+use the `grid artifacts --bucket_path <run or experiment>`.
+
+It will display a table showing where each experiment stored its artifacts:
```

Comment:
Can we clarify/explicitly name what 'it' refers to here?

Reply:
@RobertLaurella It should be something along the lines of "the output of that command will display a table showing where each experiment stored its artifacts" 
What do you think?

Reply:
I shortened it to make it simpler - I'm happy with how it reads now if this makes sense & is accurate!